static/src/js/components/CollectionDetails/CollectionDetailsBody.scss

```diff
@@ -71,6 +71,11 @@
     }
   }
 
+  &__cloud-access-title {
```

Comment:
What is this class being used for? I dont remember seeing anythign with a border-top in the UI

Reply:
That is being used by the header `Cloud Access` to keep the divisor between the cloud content and the related url section
![Screenshot 2023-09-14 at 7 57 17 PM](https://github.com/nasa/earthdata-search/assets/34591886/5a9e8074-6b65-4053-ae67-adb76a3276c7)